tests/integration/migrator/migrator_deploy_contract_cheatcode_test.py

```diff
@@ -0,0 +1,27 @@
+from pathlib import Path
+
+from protostar.migrator import Migrator
+
+
+async def test_deploy_contract(
```

Comment:
Can we include a test deploy -> interact with deployed contract on devnet?

Reply:
It will be possible after adding `call` cheatcode. I created a separate task for that.

Reply:
Can't we do the call using for example starknet.py or something?

Reply:
It must go through `GatewayFacade` to generate `StarknetInteractions`. In the gateway we can use starknet.py but this is not the scope of this PR.

Reply:
I would like to just test if contract is deployed and can be interacted with, I don't care about proper interaction logs. Without this our test are prone to false positives. Please add it.

Reply:
I'll add an interaction on the test level.

Reply:
> Without this our test are prone to false positives.
Actually, they are not.

These tests check the response from devnet.
e.g. `["code"] == "TRANSACTION_RECEIVED"`

@mkaput Should I call the devnet with Starknet.py or leave checking devnet response?

Reply:
@kasperski95 It shouldn't be "accepted on L2" ?